android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -45,6 +46,8 @@ class AppLovinMAXAdView
     @Nullable
     private Map<String, Object> localExtraParameters;
 
+    private static final Map<String, MaxAdView> ADVIEW_INSTANCES = new HashMap<>( 2 );
```

Comment:
I like this name better :)

Reply:
But `private static final` fields in Java are _always_ declared at top of file, can we look through the rest of your PR to see where conventions are not being met?

Reply:
We use `UPPER_SNAKE_CASE` only for constants (immutable fields). But since this collection is mutable, we should use `adViewInstances` instead.

Reply:
Ah, right, it should be camelCase since it is technically still mutable. I'd call it `adViewInstances`.

Reply:
Renamed in iOS as well.